src/components/NavBar/ChainSwitcher.css.ts

```diff
@@ -0,0 +1,49 @@
+import { style } from '@vanilla-extract/css'
+
+import { sprinkles } from '../../nft/css/sprinkles.css'
+
+export const ChainSwitcher = style([
```

Comment:
how does light/dark mode get handled in vanilla extract?

Reply:
Colors that have dark mode and light mode variations are defined in `themeContractValues` in the file `sprinkles.css.ts`. Then there is a `darkTheme.ts` file and a `lightTheme.ts` file which provide the hex values for the colors  defined in `themeContractValues`. Then if you just use the centrally defined variable names from the sprinkles file, for example `lightGrayContainer`, it will automatically be swapped to the appropriate hex color according to the theme.